src/ImageProcessorCore/Formats/Bmp/README.md

```diff
@@ -1,8 +1,8 @@
-﻿Encoder/Decoder adapted from:
-
-https://github.com/yufeih/Nine.Imaging/
-https://imagetools.codeplex.com/
-
-TODO:
-
-- Add support for all bitmap formats.
+﻿Encoder/Decoder adapted from:

```

Comment:
Just found it... Its set to Tab size 4... Insert spaces..

I'll have to keep a closer eye on the files that get submitted. I didn't realize this file changed.